From bf6fba2d67564905b453822e632d04b21c8c5dae Mon Sep 17 00:00:00 2001
From: Santurysim <santurysim@gmail.com>
Date: Mon, 22 Aug 2022 22:23:52 +0300
Subject: [PATCH] fix input stuff

Kernel seems to add these devices directly to /dev/input, thus old rules
didn't work. New rule is location-agnostic.

Also, change permissions to these devices since Xorg requires write access
to input devices.
---
 mdev.conf | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/mdev.conf b/mdev.conf
index 0857942..6aef152 100644
--- a/mdev.conf
+++ b/mdev.conf
@@ -99,11 +99,7 @@ psaux		root:root 0660  >misc/
 rtc		root:root 0664  >misc/
 
 # input stuff
-event[0-9]+	root:input 0640 =input/
-mice		root:input 0640 =input/
-mouse[0-9]	root:input 0640 =input/
-js[0-9]		root:input 0640 =input/
-ts[0-9]		root:input 0600 =input/
+SUBSYSTEM=input;.*  root:input 0660
 
 # v4l stuff
 vbi[0-9]	root:video 0660 >v4l/
-- 
GitLab

